qualang_tools/bakery/bakery.py

```diff
@@ -37,11 +35,13 @@ def __init__(
         config,
         padding_method: str = "right",
         override: bool = False,
-        update_config: bool = True,
         baking_index: int = None,
     ):
         self._config = config
-        self.update_config = update_config
+        if baking_index:
```

Comment:
The Baking index starts from 0 or from 1?
If it starts from 0, you have a bug.
If it starts from 1, it's cool